packages/devextreme/js/ui/popup/ui.popup.js

```diff
@@ -388,6 +390,17 @@ const Popup = Overlay.inherit({
         } else if(this._$title) {
             this._$title.detach();
         }
+
+        this._toggleAriaLabel();
+    },
+
+    _toggleAriaLabel() {
+        const { title, showTitle } = this.option();
+        const shouldSetAria = showTitle && !!title;
+        const titleId = new Guid();
```

Comment:
Let's reduce copypaste below
```suggestion
        const titleId = shouldLabelTitle ? new Guid() : null;
```

Reply:
[2ea37b479fd6e58d4d2122c7b40e37e69a5268b9]